### PR TITLE
Update `preprecossing/_qc.py:describe_var`

### LINKED ISSUE #3397: `scanpy.preprocessing.describe_var` does not use f-strings
## Description

The function `describe_var` uses the replace method for strings in combination with `locals()` instead of directly f-strings as implemented for `describe_obs`, for example.

### PR DESCRIPTION
**Pull request by WeilerP**
## Description
Use f-strings instead of `replace`.

- [x] Closes #3397.
- [x] Tests included or not required because: Already covered.

- [x] Release notes not necessary because: Minor change.

### CODE REVIEW

**Comment by WeilerP** [comment]
I didn't add any release notes since the change is minor. Let me know if you want me to add some nonetheless.

**Comment by ilan-gold** [comment]
@WeilerP I think a release note is necessary here actually, given the failing tests.  If you noticed it, others might have as well.

File: src/scanpy/preprocessing/_qc.py [hunk 1]
```diff
@@ -194,26 +194,21 @@ def describe_var(
         if issparse(X):
             X.eliminate_zeros()
     var_metrics = pd.DataFrame(index=adata.var_names)
-    var_metrics["n_cells_by_{expr_type}"], var_metrics["mean_{expr_type}"] = (
+    var_metrics[f"n_cells_by_{expr_type}"], var_metrics["mean_{expr_type}"] = (
```
Comment (ilan-gold) [inline src/scanpy/preprocessing/_qc.py]:
```suggestion
    var_metrics[f"n_cells_by_{expr_type}"], var_metrics[f"mean_{expr_type}"] = (
```

**Comment by ilan-gold** [comment]
Done in https://github.com/scverse/scanpy/pull/3325

**Comment by flying-sheep** [comment]
Thanks for the PR! I happened to do the same thing in another PR one week or so earlier

**Pull request closed without merging**